### PR TITLE
add hint on ADR IDs

### PR DESCRIPTION
**Pull request by benjguin**
# Pull Request Template

## What are you trying to address

- Add a hint on how to avoid conflicts while giving IDs to new ADRs. Cf issue #1028.

## Description of new changes

- Just added a small hint in page that describes the structure of a typical ADR, including it's ascending number / ID.

## Checklist

- [x] Link to the issue you are addressing is included above
- [ ] Contribution has a [maintenance commitment](https://internal.playbook.microsoft.com/playbook/contributing/policies/maintenance/) from the responsible team's GM or GEM
  - I think this hint (small paragraph) would become part of the page and wouldn't change the way it's maintained
- [x] Markdown pages use the approved [page templates](https://internal.playbook.microsoft.com/playbook/contributing/page-types/)
- [ ] [Contribution checklist](https://internal.playbook.microsoft.com/playbook/contributing/checklist/) is complete

  All content contributions must:
  
  - [ ] Have a maintenance commitment from the owning team's General Manager (public content) or General Engineering Manager (GEM).
  - [x] Be based on knowledge and code validated with customers.
  - [x] Have significant unique value.
  - [x] No links to personal GitHub repos without approval from an appropriate GM.§
  
  *§ We want no appearance of a conflict of interest &nbsp; the playbook is not a vehicle for promoting employee accounts. Exceptions would be links to high quality repos unrelated to the IS team.*
  
  Each Markdown file must have:
  
  - [ ] An aggregate Acrolinx score of 80 or higher† (or playbook PO provides waiver).
    - I could not connect the side bar. Per `assets/ACROLINX.md` I tried to connect with `https://microsoft.solution.acrolinx.sh/` and I got this message back: "We couldn't connect to Acrolinx. Check that the URL is correct. If you still can't connect, contact your Acrolinx administrator."
  - [ ] An Acrolinx clarity rating score is 80+ (or playbook PO provides waiver).
  - [x] No [disallowed content](./policies/disallowed-content.md).‡
  - [ ] The file owner's Microsoft alias listed in the document's [`ms.author`](https://review.learn.microsoft.com/help/platform/metadata-all-attributes?branch=main) metadata attribute.
    - There seems to be no `ms-author` for this page, but that was already the case (legacy).
  
  *† MS Learn specifies 80 as the minimum Acrolinx score.<br/>
  ‡ Be sure to confirm that images and videos don't mention customers, internal jargon, etc.*
  
  ## Public checklist
  
  In addition to the global checklist, public contributions must:
  
  - [x] Not use Microsoft-internal acronyms, team names, or other jargon unless the article is explicitly describing internal teams or processes.
  - [ ] Be reviewed and validated by at least two engineers from a different team.
  - [x] Be validated by more than one customer
  - [ ] Be reviewed and OK'd by the owning team's General Manager (GM).
    - please let me know if this is required for such a small and straight forward contribution
  - [ ] Be reviewed and OK'd by the SolutionOps GM (or delegate).
    - please let me know if this is required for such a small and straight forward contribution

- [x] Markdown pages are assigned only to `internal` or `public` ring (`cse` and `microsoft` rings are no longer supported)

### CODE REVIEW

**Review by TessFerrandez — APPROVED** [review]
LGTM!

File: docs/design/design-reviews/decision-log/README.md [hunk 1]
```diff
@@ -46,7 +50,7 @@ An architecture decision record has the structure
 
     > *Due to the microservices design of the platform, we need to ensure consistency of logging throughout each service so tracking of usage, performance, errors etc. can be performed end-to-end. A single logging/monitoring framework should be used where possible to achieve this, whilst allowing the flexibility for integration/export into other tools at a later stage. The developers should be equipped with a simple interface to log messages and metrics.*
 
-    *If the development team had a data-driven approach to back the decision, i.e. a study that evaluates the potential choices against a set of objective criteria by following the guidance in [Trade Studies](../trade-studies/README.md), the study should be referred to in this section.*  
+    *If the development team had a data-driven approach to back the decision, i.e. a study that evaluates the potential choices against a set of objective criteria by following the guidance in [Trade Studies](../trade-studies/README.md), the study should be referred to in this section.*
```
Comment (tompaana) [inline docs/design/design-reviews/decision-log/README.md:53]:
```suggestion
    *If the development team had a data-driven approach to back the decision, i.e., a study that evaluates the potential choices against a set of objective criteria by following the guidance in [Trade Studies](../trade-studies/README.md), the study should be referred to in this section.*
```

**Review by tompaana — APPROVED** [review]
Approving with a suggestion.

**Comment by benjguin** [comment]
@tompaana @TessFerrandez thanks a lot for the review. 
All looks good but a permission issue for MegaLinter

```log
❌ Linted [MDLINKCHECK] files with [linkcheck]: Found 1 error(s) - (9.61s) (expand for details)
  --Error detail:
  time="2024-02-20T16:47:59Z" level=info msg="Starting linkcheck"
  time="2024-02-20T16:47:59Z" level=info msg="Context first argument .github/ISSUE_TEMPLATE/bug_report.md"
  time="2024-02-20T16:47:59Z" level=info msg="extracting markdown files from list"
  time="2024-02-20T16:47:59Z" level=info msg="Starting to process links"
  time="2024-02-20T16:47:59Z" level=info msg="Loading cache data from /github/workspace/megalinter-reports/linchcheck-cache"
  time="2024-02-20T16:47:59Z" level=error msg="File /github/workspace/megalinter-reports/linchcheck-cache does not exist, skipping cache loadstat /github/workspace/megalinter-reports/linchcheck-cache: no such file or directory"
  time="2024-02-20T16:48:09Z" level=info msg="Went through 257 files"
  time="2024-02-20T16:48:09Z" level=info msg="****************************"
  time="2024-02-20T16:48:09Z" level=info msg="Results for file /github/workspace/docs/documentation/recipes/deploy-docfx-azure-website.md"
  time="2024-02-20T16:48:09Z" level=info
  time="2024-02-20T16:48:09Z" level=info msg="Line 58 link https://stackoverflow.com/questions/[64](https://github.com/microsoft/code-with-engineering-playbook/actions/runs/7976990382/job/21778826318?pr=1027#step:4:65)309200/is-the-custom-domain-verification-shared-across-an-azure-subscription status 403"
  time="2024-02-20T16:48:09Z" level=info
  time="2024-02-20T16:48:09Z" level=info msg="Saving cache to path /github/workspace/megalinter-reports"
  time="2024-02-20T16:48:09Z" level=fatal msg="ERROR: 1 links check failed, please check the logs"
  
  [GitHub Status Reporter] Error posting Status for MDLINKCHECKwith linkcheck: 403
  GitHub API response: {"message":"Resource not accessible by integration","documentation_url":"https://docs.github.com/rest/commits/statuses#create-a-commit-status"}
  [GitHub Status Reporter] Error posting Status for SPELLwith cspell: 403
  GitHub API response: {"message":"Resource not accessible by integration","documentation_url":"https://docs.github.com/rest/commits/statuses#create-a-commit-status"}
  [flavors] You could have same capabilities but better runtime performances if you request a new MegaLinter flavor.
  [flavors] Use the following link to request the new flavor: https://github.com/oxsecurity/megalinter/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=Request%20new%20MegaLinter%20flavor&body=MegaLinter%20would%20run%20faster%20on%20my%20project%20if%20I%20had%20a%20flavor%20containing%20the%20following%20list%20of%20linters%3A%0A%0AACTION_ACTIONLINT%2C%20MARKDOWN_MARKDOWNLINT%2C%20MARKDOWN_MARKDOWN_TABLE_FORMATTER%2C%20MDLINKCHECK_LINKCHECK%2C%20SPELL_CSPELL%2C%20YAML_PRETTIER%2C%20YAML_YAMLLINT%0A%0AWould%20it%20be%20possible%20to%20create%20one%20%3F%20Thanks%20%3Arelaxed%3A
  [GitHub Comment Reporter] Unable to post pull request comment: 403 {"message": "Resource not accessible by integration", "documentation_url": "https://docs.github.com/rest/issues/comments#create-an-issue-comment"}.
  To enable this function, please add permissions in your Github Actions Workflow:
  permissions:
    issues: write
    pull-requests: write
  ❌ Error(s) have been found during linting
  To disable linters or customize their checks, you can use a .mega-linter.yml file at the root of your repository
  More info at https://megalinter.io/7.1.0/configuration/
```

I'm not sure I can do anything about it.

**Review by shiranr — APPROVED** [review]

**Pull request merged**